.ci/build.sh

```diff
@@ -70,6 +71,15 @@ echo "Changed files: $changed_files"
 for chart in $charts; do
   chart=${chart:2}
   chart_changed=false
+
+  # Run go test if there are chart unit tests
+  if [[ -f "$chart/test/go.mod" ]]; then
+    pushd "$chart/test"
+    echo "----> Running unit tests for ${chart}"
+    go mod download
```

Comment:
explicitly downloading the modules is not necessary, it will be done automatically with `go test`